iceoryx2-bb/log/src/lib.rs

```diff
@@ -185,6 +187,44 @@ pub enum LogLevel {
     Fatal = 5,
 }
 
+impl LogLevel {
+    fn from_str_fuzzy(s: &str) -> LogLevel {
+        match s.to_lowercase().as_str() {
+            "trace" => LogLevel::Trace,
+            "debug" => LogLevel::Debug,
+            "info" => LogLevel::Info,
+            "warn" => LogLevel::Warn,
+            "error" => LogLevel::Error,
+            "fatal" => LogLevel::Fatal,
+            _ => {
+                println!(
+                    "Invalid value for 'IOX_LOG_LEVEL' environment variable!\
+                \nFound: {:?}\
+                \nAllowed is one of: fatal, error, warn, info, debug, trace\
+                \nSetting log level as : {:?}",
+                    s, DEFAULT_LOG_LEVEL
+                );
+                DEFAULT_LOG_LEVEL
```

Comment:
At this location we do not know which log level the user has chosen as fallback, the default one or a custom one. A workaround could be to pass the fallback log level to this function or just skip the actual value and print that the fallback log level is used.